src/wp-includes/global-styles-and-settings.php

```diff
@@ -336,6 +408,10 @@ function wp_add_global_styles_for_blocks() {
 			}
 		}
 	}
+
+	if ( ! wp_should_load_separate_core_block_assets() ) {
+		wp_enqueue_style( 'global-styles-blocks' );
+	}
```

Comment:
I'm not sure if this should be enqueued here (or registered above) as it's a rather large change for a public function that just used to add styles.

Reply:
Hmm good point, I'll see if I can find a more appropriate place.